ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1271,3 +1271,59 @@ def test_tensorflow_linspace(
         axis=axis,
         on_device=on_device,
     )
+   
+
+# tile
+@st.composite
+def _multiple_shape_helper(draw):
+    input_dtype, input_array, input_shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
+            ret_shape=True
+        ))
+    
+    # input = dtypes_and_input[1][0]
+    input_dims = len(input_shape)
+    
+    shape = draw(helpers.get_shape(
+        min_num_dims=1, max_num_dims=1,
+        min_dim_size=input_dims, max_dim_size=input_dims
+    ))
+    dt_n_multiples = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["int32", "int64"],
+            shape=shape))  
+    # return dtypes_n_input, dt_n_multiples
+    return input_dtype, input_array, dt_n_multiples
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.tile",
+    all_arguments=_multiple_shape_helper()
+)
+def test_tensorflow_tile(
+    *,
+    all_arguments,
+    num_positional_args,
+    as_variable,
+    native_array,
+    frontend,
+    fn_tree,
+    on_device
+):
+    # dtypes_and_input, dt_and_multiples = all_arguments
+    input_dtype, input_matrix, dt_and_multiples = all_arguments
+    # real_dtype, input_matrix = dtypes_and_input
+    dt_mul, multiples = dt_and_multiples
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        input=input_matrix,
```

Comment:
Right now there is an error because `input_dtype` and `input_matrix` are lists of length 1 containing the arrays we actually need. So you need to get `input_dtype[0]` and `input_matrix[0]`.